Overview.bs

```diff
@@ -275,11 +275,11 @@ needed to render a subset of:
 
 supported by the original font. When a subsetted font is used to render text using any combination of the subset
 codepoints, [[open-type/featuretags|layout features]], or [[open-type/otvaroverview#terminology|design-variation space]]
-it should render identically to the original font. This includes rendering with the use of any optional typographic
+it must render identically to the original font. This includes rendering with the use of any optional typographic
```

Comment:
For now can you leave this and the the one below as should's. I'll be spending some time updating this section (and the encoder section) as discussed in todays call and during those edits we can consider whether to make this a should or must.

As the spec is currently written, font subset is used pretty liberally throughout and I want to avoid unintentionally introducing normative requirements whenever "font subset" is referenced. Roughly, I plan to have the encoder section introduce the normative rendering equivalence requirement (conditionally when an encoder is making a neutral encoding). That way we can very specifically lay out where rendering equivalence is required and what exactly is the "original font".

Reply:
I figure that the in-meeting discussion of that stuff supersedes these editorial suggestions.